core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -620,7 +628,11 @@ public MetricKey build() {
           .build();
   public static final MetricKey MASTER_UFS_JOURNAL_INITIAL_REPLAY_TIME_MS =
       new Builder("Master.UfsJournalInitialReplayTimeMs")
-          .setDescription("The process time of the ufs journal initial replay")
+          .setDescription("The process time of the ufs journal initial replay."
+              + "Only valid when ufs journal is used."
+              + " It records the time it took for the very first journal replay. "
+              + "It can be used to to determine if journal replay is the reason "
```

Comment:
This sounds very specific.
Can you say something like above: "Use this metric to monitor when your master boot-up time is high."